internal/query/projection/limits.go

```diff
@@ -104,11 +111,26 @@ func (p *limitsProjection) reduceLimitsReset(event eventstore.Event) (*handler.S
 	if err != nil {
 		return nil, err
 	}
-	return handler.NewDeleteStatement(
-		e,
-		[]handler.Condition{
-			handler.NewCond(LimitsColumnInstanceID, e.Aggregate().InstanceID),
-			handler.NewCond(LimitsColumnResourceOwner, e.Aggregate().ResourceOwner),
-		},
-	), nil
+	cond := []handler.Condition{
+		handler.NewCond(LimitsColumnInstanceID, e.Aggregate().InstanceID),
+		handler.NewCond(LimitsColumnResourceOwner, e.Aggregate().ResourceOwner),
+	}
+	if e.Properties == nil {
+		return handler.NewDeleteStatement(e, cond), nil
```

Comment:
This situation should never happend right? Otherwise it should never create such an event, so I would handle it as an error.

Reply:
Or as a NoOp statement as with the length check below

Reply:
We just don't provide an API for this (yet)

Reply:
You have a ResetLimits call, which doesn't reset all limits and you have a event which would delete the limits, aka reset, if you provide no properties, which should technically never happen?

Reply:
obsolete, as we have no reset event anymore